src/pages/community/download.md

```diff
@@ -31,10 +31,11 @@ All release artifacts should be checked for tampering using GPG or SHA-512.
+| v1.2.0  | 2023-02-02   | [Download](https://www.apache.org/dyn/closer.lua/yunikorn/1.2.0/apache-yunikorn-1.2.0-src.tar.gz) <br />[Checksum](https://downloads.apache.org/yunikorn/1.2.0/apache-yunikorn-1.2.0-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/yunikorn/1.1.0/apache-yunikorn-1.2.0-src.tar.gz.asc)                                                                      | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-1.2.0/images/sha256-c3b1a7b2cfec3f3560415519278cc4d94748f0f60ee80dfaf23fcc22dbb8b8e5) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-1.2.0/images/sha256-7f18fcd080640974ae586d30eda009daf0ad93fa22ada66b0a337ac3fb46b7ac) <br />[web](https://hub.docker.com/layers/apache/yunikorn/web-1.2.0/images/sha256-706a2895461623f20d4102f0132d66dade9c15edf5cad40065506a4d70c32576) <br />[scheduler plugin](https://hub.docker.com/layers/apache/yunikorn/scheduler-plugin-1.0.0/images/sha256-d946495946b89d03f7a8f786702d4b350a93f74d52e50bebb6b2bbdcb8e911a4)        | [Announcement](/release-announce/1.2.0)  |
 | v1.1.0  | 2022-09-08   | [Download](https://www.apache.org/dyn/closer.lua/yunikorn/1.1.0/apache-yunikorn-1.1.0-src.tar.gz) <br />[Checksum](https://downloads.apache.org/yunikorn/1.1.0/apache-yunikorn-1.1.0-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/yunikorn/1.1.0/apache-yunikorn-1.1.0-src.tar.gz.asc)                                                                      | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-1.1.0/images/sha256-5a45cede355b4c1d0016ba81b317e12a7608ac5de4779892f8c7fa53adf5d739) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-1.1.0/images/sha256-4389c126f252671e55bdac16e1bcfe7f83ef4ea7c3e83d333c81508920da825c) <br />[web](https://hub.docker.com/layers/apache/yunikorn/web-1.1.0/images/sha256-3f3075161283d8a78f4849f8163104d7db3e7bd3a467163729fb401421ac670f) <br />[scheduler plugin](https://hub.docker.com/layers/apache/yunikorn/scheduler-plugin-1.0.0/images/sha256-f7b2a186b3088e269842c415e1fe1c2afa8835e24a98fa85097e6be5c234712b) | [Announcement](/release-announce/1.1.0)  |
```

Comment:
These 3 links needs to point to archive like the 1.0.0 links:
```
https://archive.apache.org/dist/yunikorn
```
We also need to fix the plugin image link that points to 1.0.0 this is the correct 1.1.0: 
```
https://hub.docker.com/layers/apache/yunikorn/scheduler-plugin-1.1.0/images/sha256-b182ec4d08c590cd179b811fe68dffa2d58dccc9b8568f108ad35af0ed62a4c1?context=explore
```